packages/terra-functional-testing/tests/fixtures/reporters/test-data-1.json

```diff
@@ -0,0 +1,75 @@
+{
+  "title": "(root)",
```

Comment:
Should this be the spec file that was ran?

Reply:
No, this was generated directly by the test runner and copy pasted into this file. The root suite is always assigned '(root)':

https://github.com/webdriverio/webdriverio/blob/master/packages/wdio-reporter/src/index.ts#L58